sdk/plugin_interface/xrcconv.cpp

```diff
@@ -413,7 +413,10 @@ void ObjectToXrcFilter::AddWindowProperties()
 	if (!m_obj->IsNull(_("enabled")) && m_obj->GetPropertyAsInteger(_("enabled")) == 0) {
 		AddProperty(_("enabled"), _("enabled"), XRC_TYPE_BOOL);
 	}
-	if ( !m_obj->IsNull( _( "focused" ) ) )
+	if (!m_obj->IsNull(_("create_children")) && m_obj->GetPropertyAsInteger(_("create_children")) == 0) {
+        AddProperty(_("create_children"), _("create_children"), XRC_TYPE_BOOL);
+    }
```

Comment:
This is useless because the XRC "language" doesn't define this property, the XRC handlers shipped with wxWidgets simply won't read this property